src/Components/PeoplePage/PeoplePage.jsx

```diff
@@ -0,0 +1,53 @@
+import React, { useState, useEffect } from 'react';
+import { getPeople } from '../../Api';
+import { PersonRow } from '../PersonRow';
+import './peoplePage.css';
+
+export const PeoplePage = () => {
+    const [people, setPeople] = useState([]);
+
+    const findParents = (array) => {
+        const result = array.map((person) => {
+            person.mother = array.find(({ name }) => name === person.motherName);
+            person.father = array.find(({ name }) => name === person.fatherName);
+
+            return person;
+        });
+
+        setPeople(result);
+    };
+
+    useEffect(() => {
+        getPeople()
+            .then(findParents);
+    }, []);
+
+    return (
+        <>
+            <h2>People page</h2>
+
+            <table className="table">
```

Comment:
Let's create `PeopleTable` component for the table (as task description requires)